### PR TITLE
Migrate group functions from Carbon

### PR DESCRIPTION
**Pull request by splatteredbits**
I need a version of Add-CGroupMember that prevents local nested groups from getting added to a group.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,47 @@
...
+* Rename usages of `Install-CGroup`, `Add-CGroupMember`, and `Remove-CGroupMember` function's `Members` parameter to
+  `Member`.
+* Remove usages of `Install-CGroup` function's `PassThru` switch. Use `Get-LocalGroup` instead.
+* Rename usages of `Test-CGroupMember` function's `GroupName` parameter to `Name`.
+* `Install-CGroup` and `Add-CGroupMember` no longer support adding a local group to a group and writes an error instead.
+  Previously, local built-in groups were allowed to be added because the underlying Windows API allowed them, but
+  functionally, Windows ignores local groups in a local group. Review usages. To check if a group is a local group, use
+  `Resolve-CIdentity` and if the `Type` property on the returned object is `Alias` and the group is not a domain group,
+  it will cause an error.
+* Rename usages of `Add-CGroupMember` to `Install-CLocalGroupMember`.
```
Comment (decoyjoe) [inline CHANGELOG.md:26]: I would prefer to keep the verb `Add-`

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,47 @@
...
+* Rename usages of `Test-CGroupMember` to `Test-CLocalGroupMember`.
+* Rename usages of `Uninstall-CGroup` to `Uninstall-CLocalGroup`.
+
+### Added
+
+* `Get-CLocalGroup` for getting local groups. Has support for non-wildcard lookups.
+* `Install-CLocalGroup` for installing local groups.
+* `Install-CLocalGroupMember` for adding accounts to local groups.
+* `Test-CLocalGroup` for testing if local groups exist.
+* `Test-CLocalGroupMember` for testing if accounts are
```
Comment (decoyjoe) [inline CHANGELOG.md]: Unfinished sentence.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,47 @@
+<!--markdownlint-disable MD012 no-multiple-blanks-->
+<!--markdownlint-disable MD024 no-duplicate-heading-->
 
 # Carbon.Accounts PowerShell Module Changelog
 
+## 1.1.0
```
Comment (decoyjoe) [inline CHANGELOG.md]: Version is `2.0.0`

File: Carbon.Accounts/Functions/Install-CLocalGroupMember.ps1 [hunk 4]
```diff
@@ -0,0 +1,91 @@
+
+function Install-CLocalGroupMember
+{
+    <#
+    .SYNOPSIS
+    Adds a users or groups to a local group.
```
Comment (decoyjoe) [inline Carbon.Accounts/Functions/Install-CLocalGroupMember.ps1]: Typo: `Adds a users`

File: Carbon.Accounts/Functions/Uninstall-CLocalGroup.ps1 [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    #>
+    [CmdletBinding(SupportsShouldProcess=$true)]
+    param(
+        # The name of the group to remove/uninstall.
+        [Parameter(Mandatory)]
+        [ValidateNotNullOrEmpty()]
+        [String] $Name
+    )
+
+	Set-StrictMode -Version 'Latest'
```
Comment (decoyjoe) [inline Carbon.Accounts/Functions/Uninstall-CLocalGroup.ps1]: Indented too far.

**Review by decoyjoe — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (splatteredbits) [reply CHANGELOG.md:26]: Typically, commands with an `Add` or `Remove` verb write an error if what is being added is already present or what is being removed isn't there. So, I switched to using `Install` and `Uninstall` verbs, which is Carbon's pattern for idempotent operations.

**Pull request merged**